inc/admin/namespace.php

```diff
@@ -538,7 +583,7 @@ function handle_regenerate() {
 	$id = absint( $_GET['id'] );
 	check_admin_referer( 'rest-oauth2-regenerate:' . $id );
 
-	if ( ! current_user_can( 'edit_post', $id ) ) {
+	if ( ! current_user_can( 'edit_posts', $id ) ) {
```

Comment:
This check was correct before; `edit_post` is a meta-cap, which uses the `$id` parameter to map to an internal capability based on the post type of `$id` (allowing other plugins to change the OAuth permissions more easily).